templates/lxd/manage.html

```diff
@@ -0,0 +1,392 @@
+        <div class="col-3 col-medium-2 p-image-wrapper">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/9083ef88-bolt.png",
+          alt="Bolt logo",
+          width="107",
+          height="40",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+          }}
+        </div>
+        <div class="col-6 col-medium-4 p-section--shallow">
+          <p><a href="https://www.puppet.com/docs/bolt/latest/bolt.html">Bolt</a> is an open source orchestration tool
+            that automates the manual work it takes to maintain your infrastructure.</p>
+          <p>The LXD transport allows for interacting with LXD instances.</p>
+          <p>For more information, see the <a
+              href="https://www.puppet.com/docs/bolt/latest/bolt_transports_reference.html#lxd">Puppet Bolt
+              Documentation on LXD</a>.</p>
+          <p>To manage LXD in Puppet Bolt, you need a LXD server (see <a
+              href="https://canonical-lxd.readthedocs-hosted.com/en/latest/getting_started/">Getting started</a>).</p>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row question-heading" id="packer">
+        <div class="col-3 col-medium-2 p-image-wrapper">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/91e0050c-packer.png",
+          alt="Packer logo",
+          width="132",
+          height="49",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+          }}
+        </div>
+        <div class="col-6 col-medium-4 p-section--shallow">
+          <p><a href="https://www.packer.io/">Packer</a> is an open source tool for creating identical machine images
+            for multiple platforms.</p>
+          <p>With Packer's LXD builder, it's possible to re-use your existing cloud image building pipeline and with the
+            LXD builder, build a LXD container image.</p>
+          <p>Take a look at the <a href="https://www.packer.io/docs/builders/lxd">Packer documentation on LXD</a> for
+            more information.</p>
+          <p>Additionally our guides about <a
+              href="https://canonical-lxd.readthedocs-hosted.com/en/latest/images/">images</a> and <a
+              href="https://canonical-lxd.readthedocs-hosted.com/en/latest/howto/instances_configure/">instance
+              configuration</a> might contain useful information regarding image choice, configuration options etc.</p>
+          <p>To create LXD images in Packer, you need a LXD server (see <a
+              href="https://canonical-lxd.readthedocs-hosted.com/en/latest/getting_started/">Getting started</a>).</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--white is-deep">
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <h2><a href="/lxd/contact-us" aria-controls="lxd-install-modal">Talk to our LXD experts&nbsp;&rsaquo;</a></h2>
```

Comment:
`/lxs/contact-us` doesn't exist, did you mean `/contact-us` as in `/lxd/install` page?